backends/platform/3ds/sprite.cpp

```diff
@@ -20,17 +20,23 @@
  *
  */
 
+#include "osystem.h"
```

Comment:
```suggestion
#include "backends/platform/3ds/osystem.h"
```

Reply:
There are a few other instances like this where the full path isn't used. Do you want me to change those as well?

Reply:
Your choice.